scripts/seek/runParallelJobs.py

```diff
@@ -54,7 +54,7 @@ def jobThread(work_queue, tid=0, returnCodeQueue=None, outputQueue=None):
     except queue.Empty:
         pass
     return
-    
+
```

Comment:
There is syntax error on lines with:
`cmd := work_que` or  `job := work_queue.get(block=False):`

Reply:
So this is a new syntax added in Python 3.8. Can you update your conda python version?

Reply:
Ok, that helped